script.js

```diff
@@ -0,0 +1,125 @@
+const overlay = document.querySelector('#myNav');
+const links = document.querySelector('.menu-link');
+const links1 = document.querySelector('.menu-link1');
+const links2 = document.querySelector('.menu-link2');
+links.addEventListener('click', () => {
+  if (overlay.style.display === 'block') {
+    overlay.style.width = '100%';
+  } else {
+    overlay.style.width = '0%';
+  }
+});
+links1.addEventListener('click', () => {
+  if (overlay.style.display === 'block') {
+    overlay.style.width = '100%';
+  } else {
+    overlay.style.width = '0%';
+  }
+});
+links2.addEventListener('click', () => {
+  if (overlay.style.display === 'block') {
+    overlay.style.width = '100%';
+  } else {
+    overlay.style.width = '0%';
+  }
+});
+
+// Data array
+const Data = [
+  {
+    source: 'images/car3.jpg',
+    alt: 'car-image',
+    class: 'Conducted by Travor',
+    subtitle: 'Travel Plan',
+    description: 'A travel plan is a package of actions designed by a workplace, school or other organisation.',
+  },
+  {
+    source: 'images/catering.jpg',
+    alt: 'food-image',
+    class: 'Conducted by Grace',
+    subtitle: 'Catering Service',
+    description: 'The full service means you will have kitchen staff, bartenders, servers, and the front of the.',
+  },
+  {
+    source: 'images/baby.jpg',
+    alt: 'car-image',
+    class: 'Conducted by Mariam',
+    subtitle: 'Babysitting',
+    description: 'the work of taking care of someones baby or child while that person is out, often as a paid job.',
+  },
+  {
+    source: 'images/loundary2.jpg',
+    alt: 'loundary',
+    class: 'Conducted by Joseph',
+    subtitle: 'Laundry',
+    description: 'Laundry refers to the clothes and bedding that you can wash in your washing machine.',
+  },
+  {
+    source: 'images/car1.jpg',
+    alt: 'car-image',
+    class: 'Conducted by Lawrence',
+    subtitle: 'Hire Driver',
+    description: 'Rent A Driver Uganda offers professional and experienced private drivers in Uganda.',
+  },
+  {
+    source: 'images/bar1.jpg',
+    alt: 'drinks-image',
+    class: 'Conducte by Flaviour',
+    subtitle: 'Bar & Drink',
+    description: 'Heres a list that every bartender should know to be successful at their job..',
+  },
+];
+const section = document.querySelector('.services');
+Data.forEach((item, index) => {
+  const container = document.createElement('div');
+  container.className = 'container';
+  const image = document.createElement('img');
+  image.src = Data[index].source;
+  image.alt = Data[index].alt;
+  container.appendChild(image);
+  section.appendChild(container);
+  const services = document.createElement('div');
+  services.className = 'services-item';
+  container.appendChild(services);
+  const heading = document.createElement('h2');
+  heading.textContent = Data[index].subtitle;
+  services.appendChild(heading);
+  const paragraph = document.createElement('p');
+  paragraph.textContent = Data[index].description;
+  services.appendChild(paragraph);
+});
+
```

Comment:
- [ ] Kindly decrease the height for the servicer container since it is very big in your design so descrease it to be the same with the oriignal design ✅✅✅

-[ ] please use the same backgorund color to this section since it is white in your case but it is different in the design 


- Your design
![image](https://user-images.githubusercontent.com/58553711/181908302-801b8167-986d-4df8-b4db-26387570acfb.png)